templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,22 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+if (Test-Path -path "<%= windows_path %>") {
+  Write-Debug "Warning: copying over '<%= windows_path %>'"
+}
+$origin = Join-Path -path "$toolsDir" -childpath "<%= dir.gsub('/', '\\') %>"
+Copy-Item -path "$origin\*" -destination "<%= windows_path.gsub('/', '\\') %>" -Recurse -Force
+<%- end -%>
+
+<%- filenames = get_files.map {|f| f.path} -%>
+<%- configfilenames = get_configfiles.map {|cf| cf.path} -%>
+<%- (filenames + configfilenames).each do |file| -%>
+  <%- windows_path = platform.convert_to_windows_path(file) -%>
+if (Test-Path -path "<%= windows_path %>") {
+  Write-Debug "Warning: copying over '<%= windows_path %>'"
+}
+$origin = Join-Path -path "$toolsDir" -childpath "<%= file.gsub('/', '\\') %>"
+Copy-Item -path "$origin" -destination "<%= windows_path.gsub('/', '\\') %>" -Force
+<%- end -%>
```

Comment:
shoot, I realized this isn't doing what we want....... I haven't figured out why yet.

it should be copying 
```
$ ls /cygdrive/c/ProgramData/chocolatey/lib/pl-boost/tools/cygdrive/c/tools/pl-build-tools/
bin  include  lib  share
```

out to 
```
$ ls /cygdrive/c/tools/pl-build-tools/
$Recycle.Bin  b2.exe  bin  bjam.exe  crash_dumps  cygwin64  include  lib  share
```
but, as you can see, we're getting extra things I what shouldn't be there, namely `cygwin64`